gubernator/github/handlers.py

```diff
@@ -83,15 +84,16 @@ def post(self):
             repo=repo, number=number, event=event, body=body, **kwargs)
         webhook.put()
 
+        # Defer digest updates, so they'll retry on failure.
```

Comment:
"Retrying a failed task

If a push task request handler returns an HTTP status code outside the range 200–299, or fails to return any response before the task deadline occurs, App Engine retries the task until it succeeds. The system backs off gradually to avoid flooding your application with too many requests, but schedules retry attempts for failed tasks to recur at a maximum of once per hour."